change/@fluentui-react-progress-bf837b7d-856a-4256-94fc-ebd888027c86.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

This needs to be `prerelease`:
```suggestion
  "type": "prerelease",
```

You'll also need to change the beachball settings in **react-progress/package.json** to:
```
  "beachball": {
    "disallowedChangeTypes": [
      "major",
      "minor",
      "patch"
    ]
  }
```

---

(This was caused by a bug in the create-package script that I've since fixed in #24763)